src/models/ErrorHandler.js

```diff
@@ -0,0 +1,32 @@
+import { Logger } from 'zos-lib'
+
+const log = new Logger('Error')
+
+export default class ErrorHandler {
```

Comment:
I guess we'll always disagree on using objects vs plain functions for this kind of stuff :-P

Reply:
We should decide a pattern and follow a single one through all the app